pulp_smash/tests/rpm/utils.py

```diff
@@ -76,6 +76,17 @@ def check_issue_2798(cfg):
             selectors.bug_is_untestable(2798, cfg.version))
 
 
+def check_issue_1282(cfg):
+    """Return true if `Pulp #1282`_ affects the targeted Pulp system.
+
+    :param pulp_smash.config.PulpSmashConfig cfg: The Pulp system under test.
+
+    .. _Pulp #1282: https://pulp.plan.io/issues/1282
+    """
+    return (cfg.version < Version('2.14') or
+            selectors.bug_is_untestable(1282, cfg.version))
+
+
```

Comment:
More for my edification than anything, why do you sometimes define a "check_issue" function and sometimes not? 
Is it simply because this has two conditions where either might cause the bugfix to be untestable (because it doesn't yet exist)?